honeybadgermpc/router.py

```diff
@@ -12,7 +12,7 @@ def simple_router(n):
 
     def make_send(i):
         def _send(j, o):
-            logging.debug('SEND %8s [%2d -> %2d]' % (o, i, j))
```

Comment:
Would making this an `info` debugging call be able to get performance improvements since our tests run with `DEBUG` logging?